be/src/storage/primary_key_encoder.cpp

```diff
@@ -354,16 +355,16 @@ Status PrimaryKeyEncoder::create_column(const vectorized::VectorizedSchema& sche
 
 typedef void (*EncodeOp)(const void*, int, std::string*);
 
-static void prepare_ops_datas(const vectorized::VectorizedSchema& schema, const vectorized::Chunk& chunk,
-                              std::vector<EncodeOp>* pops, std::vector<const void*>* pdatas) {
-    int ncol = schema.num_key_fields();
+static void prepare_ops_datas(const vectorized::VectorizedSchema& schema, const std::vector<ColumnId>& sort_key_idxes,
```

Comment:
Add context to describe the bug and solution.

Reply:
ok, added.